web/app/dao/UserDAO.java

```diff
@@ -37,8 +37,8 @@
 			"VALUES(?, ?, SHA1(?), ? , 'SHA1', 'default')";
 
 	private final static String CREATE_LDAP_USER = "INSERT INTO users " +
-			"(name, username, email, department_number, password_digest_type) " +
-			"VALUES(?, ?, ?, ?, 'SHA1')";
+			"(name, username, email, department_number, password_digest_type, authentication_type) " +
+			"VALUES(?, ?, ?, ?, 'SHA1', 'LDAP')";
```

Comment:
'SHA1' is not needed here because we don't save hash for LDAP-authenticated users